<launch>
	<!-- arguments -->
	<arg name="map_file" default="$(find rrt_planning)/maps/map.yaml"/>
	
	<!-- Nodes -->
	<node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />
	
	<node name="rviz" pkg="rviz" type="rviz" respawn="true"
	args="-d $(find rrt_planning)/rviz_config/rviz.rviz" />
	
	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
		<!-- default configuration -->
		<rosparam file="$(find rrt_planning)/config/global_planner.yaml" command="load" />
		<rosparam file="$(find rrt_planning)/config/local_planner.yaml" command="load" />
		
		<param name="base_global_planner" value="rrt_planning/ThetaStarPlanner"/>
	</node>
	
	<!-- node pkg="amcl" type="amcl" name="amcl">
		<param name="odom_frame_id"             value="odom"/> 
		<param name="base_frame_id"             value="base_footprint"/> 
		<param name="global_frame_id"           value="map"/>
		<param name="initial_pose_x"            value="0"/>
		<param name="initial_pose_y"            value="0"/>
		<param name="initial_pose_a"            value="0"/>
	</node -->
	
	<node pkg="rrt_planning" 
	type="test_node" 
	name="false_odom" 
	args="odom base_footprint 100" />
	
	<node pkg="tf" 
	type="static_transform_publisher" 
	name="false_localization" 
	args="0 0 0 0 0 0 1 map odom 100" />
	
</launch> 
